pyproject.toml

```diff
@@ -2,6 +2,7 @@
 # Only used by `hf-doc-builder´.
 line-length = 119
 target-version = ['py38']
+experimental-string-processing = true
```

Comment:
This won't have any effect, as we moved to ruff for formatting, see comment above.

Reply:
Thanks. Updated.